docs/book/advanced-guide/gcp_entrypoint.md

```diff
@@ -0,0 +1,277 @@
+# Getting started with Google Cloud
+
+This guide will take you from the default local stack to an opinionated cloud 
```

Comment:
Its not really an 'opiniated' stack. Its more of a basic stack with the least frills